packages/hint-image-optimization-cloudinary/README.md

```diff
@@ -11,9 +11,8 @@ the hint’s configuration. You can create a [free account here][signup].
 
 ## Why is this important?
 
-As of September 15th, 2017, and based on [HTTP Archive][httparchive-report],
-images account for a bit over 53% of the size of a website in average,
-or 1,810kB.
+As of June 1st, 2017, and based on [HTTP Archive][httparchive-report],
```

Comment:
```suggestion
As of June 1st, 2019, and based on [HTTP Archive][httparchive-report],
```

Reply:
🤦‍♂️

Reply:
For some reason, I can't apply suggestions... :(